### PR TITLE
Adding some go tools

### PR DESCRIPTION
**Pull request by bobthemighty**

### CODE REVIEW

File: main/gosu/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    export GOPATH="/home/builder/package"
+    go get github.com/tianon/gosu
+    go build github.com/tianon/gosu
+}
+
+package() {
+    cd "$_builddir"
+    install -Dm755 $pkgname "$pkgdir/usr/bin/$pkgname"
+}
+
```
Comment (egidijus) [inline main/gosu/APKBUILD]: do we want a sha sum of something ?

Comment (bobthemighty) [inline main/gosu/APKBUILD]: We don't download any sources explicitly, which makes that problematic.

Comment (egidijus) [inline main/gosu/APKBUILD]: YDD

Comment (egidijus) [inline main/gosu/APKBUILD]: if we shasum the output binary, does the packager care ?
that might work

File: main/gosu/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Maintainer: Bob Gregory <bob@made.com>
+pkgname=gosu
+pkgver=1.7
+pkgrel=0
+pkgdesc="tianon/gosu for Alpine"
+url="https://github.com/tianon/gosu"
+arch="x86_64"
+license="MIT"
+depends=""
+makedepends="go musl-dev linux-headers"
```
Comment (egidijus) [inline main/gosu/APKBUILD]: shouldnt go be a depends as well >? like above ?

Comment (bobthemighty) [inline main/gosu/APKBUILD]: Go is only required while building, it's not a runtime dependency.

File: main/gosu/APKBUILD (hunk 1 above)
Comment (bobthemighty) [inline main/gosu/APKBUILD]: No, it won't have it, the shas are just for source tar balls.

**Comment by egidijus** [comment]
merge - yolo

**Pull request merged**